packages/docusaurus-plugin-content-blog/src/authors.ts

```diff
@@ -148,10 +158,28 @@ ${Object.keys(authorsMap)
   return frontMatterAuthors.map(toAuthor);
 }
 
+function fixAuthorImageBaseURL(
+  authors: Author[],
+  {baseUrl}: {baseUrl: string},
+) {
+  authors.forEach((author) => {
+    if (
+      !author.imageURL ||
+      baseUrl === '/' ||
+      author.imageURL.startsWith('http')
+    ) {
+      return;
+    }
+    author.imageURL = path.join(baseUrl, author.imageURL);
```

Comment:
`path.join` is not ideal for this situation.
Separator will be different depending on OS (`/` or `\`), that's why Windows tests fail
It's preferable to use our `normalizeUrl()` util